cmd/firehose-cosmos/app_firehose.go

```diff
@@ -39,6 +39,7 @@ func init() {
 		cmd.Flags().Uint64("firehose-tracker-offset", 100, "Number of blocks for the bstream block resolver")
 		cmd.Flags().String("firehose-block-index-url", "", "If non-empty, will use this URL as a store to load index data used by some transforms")
 		cmd.Flags().IntSlice("firehose-block-index-sizes", []int{100000, 10000, 1000, 100}, "List of sizes for block indices")
+		cmd.Flags().String("firehose-rpc-head-tracker-url", "", "If non-empty, will use this URL to make RPC calls to status endpoint")
```

Comment:
Do we want to set here default value?

Reply:
Cant have a default because there's no official RPC endpoint. Plus we only need to use it when bstream is not available, so this is basically a fallback option.